Projektskabelon-master/src/main/java/FunctionLayer/OverlayMaterialCalculator.java

```diff
@@ -231,18 +239,21 @@ public  static Material screwForOverlayOneWall ( Wall wall, String overlayName)
         screwFyr.setName("BASIC SKRUE 5,0X40MM");
         screwFyr.setSize(screwFyrQuantity); //for the door - always the same size of the door
         screwFyr.setComment("Til montering af vertical framing-dør");
+        screwFyr.setPrice(LogicFacade.getPrice(screwFyr.getId()));
```

Comment:
If you set the price here, it will be calculated for the needed size, not for the available one. I calculate de available sizes first in function layer